lib/gooddata/commands/datasets.rb

```diff
@@ -27,7 +27,7 @@ def index
         # connect
         with_project do |project_id|
           Project[project_id].datasets.each do |ds|
-            puts "#{ds.uri}\t#{ds.identifier}\t#{ds.title}"
+            GoodData.logger.info("#{ds.uri}\t#{ds.identifier}\t#{ds.title}")
```

Comment:
info: Refers to 'ds' more than self (maybe move it to another class?) (FeatureEnvy) (reek)